autotest/gcore/vrt_read.py

```diff
@@ -1625,6 +1625,14 @@ def test_vrt_protocol():
         assert not gdal.Open("vrt://data/minfloat.tif?a_ullr=0,1,1,0&unscale&")
 
 
+@pytest.mark.require_proj(7, 2)
+def test_vrt_protocol_a_coord_epoch_option():
+    ds = gdal.Open("vrt://data/byte.tif?a_coord_epoch=2021.3")
+    srs = ds.GetSpatialRef()
+    assert srs.IsDynamic()
```

Comment:
byte.tif uses a projected CRS over NAD27 which is not a dynamic CRS. You should probably also add a_srs=EPSG:4326

Reply:
does that affect the test though?  (humour me 😃)

Reply:
well, this is the reason why it fails:
```
25: >       assert srs.IsDynamic()
25: E       AssertionError: assert False
```

Reply:
oh, thanks yes learning but still not looking enough 🙏